test/specs/instance.spec.ts

```diff
@@ -684,6 +684,27 @@ describe('instance', () => {
         });
     });
 
+    it('should udpate associations when given ids instead of models', async() =>
```

Comment:
Ah, I think it is the usage of `async` here, which implicitly requires the native `Promise` api and conflicts with the bluebird implementation. Removing this, should fix it.

Sry for the late response